lib/renderer.js

```diff
@@ -55,7 +68,7 @@ const renderRecipes = (recipes) => {
 }
 
 const renderGroceryList = (groceries) => {
-  JSON.parse(groceries).all_Groceries.map(grocery => {
+  JSON.parse(groceries).grocery_list.map(grocery => {
```

Comment:
On this branch, the grocery list loads on render- I think this was all it was :)